### DISCUSSION TITLE
Are there plans to migrate from SQLAlchemy 1.3 to 1.4?

### DISCUSSION THREAD

**Discussion by pajot** [openego/oeplatform#927] [Q&A] [answered]
SQLAlchemy 1.3 is a so-called maintenance release receiving only minimal fixes. [The upstream maintainers advise projects in active development to begin migrating to 1.4.](https://www.sqlalchemy.org/download.html#maintenance)

Is there a plan for the OEP to take this step?

**Comment by jh-RLI** [comment, marked as answer]
There are plans but due to lack of time and resources ATM it is not clear when this will be the case. :) :/
